features/liquidity/components/LiquidityBreakdown.tsx

```diff
@@ -73,13 +74,34 @@ const PoolHeader = ({ tokenA, tokenB, priceBreakdown }: PoolHeaderProps) => (
   </Inline>
 )
 
+const SwapFeeElem = ({
+  protocolFee,
+  lpFee,
+}: {
+  protocolFee?: number
+  lpFee?: number
+}) => (
+  <>
+    <Text variant="header">{`${(protocolFee || 0) + (lpFee || 0)}%`}</Text>
+    <Tooltip
+      label={`${lpFee || 0}% of Swap Fee goes to LP Providers (LP) and ${
```

Comment:
also this should be a check if lpFee and protocolFee is undefined, we should only use the default if the values are undefined. If the values are actually 0 we should display 0.

Reply:
Hey I can make this change, but if it's `undefined` for the older smart contracts, wouldn't the lpFee be 0.3%?

Reply:
What if they are both undefined? What do we display?

Reply:
This doesn't make sense. We should just display the safe defaulted versions if they are undefined.

Reply:
they will be either both defined or both undefined